File: Codeforces/2044B-Nomal_Problem.cpp
#include <iostream>
#include <string>

int main() {
  int t;
  std::cin >> t;

  while (t--) {
    std::string s;
    std::cin >> s;

    int n = s.length();
    std::string ans = "";
    for (int i = n - 1; i >= 0; i--) {
      if (s[i] == 'q') ans += 'p';
      else if (s[i] == 'p') ans += 'q';
      else ans += 'w';
    }

    std::cout << ans << "\n";
  }
}

File: Codeforces/2044B-Nomal_Problem.py
def solve(s, n):
    ans = ""
    for i in range(n - 1, -1, -1):
        if s[i] == "q": ans += "p"
        elif s[i] == "p": ans += "q"
        else: ans += "w"
    return ans

for _ in range(int(input())):
    s = input()
    ans = solve(s, len(s))
    print(ans)
